INFO: 2018-08-21 14:16:50,977:Dart.py:120: Logfile is /root/Dart/log/Dart-ClusterHealth-None-141650.log
INFO: 2018-08-21 14:16:51,398:Dart.py:390: No resources found.

INFO: 2018-08-21 14:16:51,400:Dart.py:418: No
INFO: 2018-08-21 14:16:51,401:Dart.py:424: kubectl get -o template po No --namespace=cloud-aster --template={{.spec.nodeName}}
INFO: 2018-08-21 14:16:51,782:Dart.py:390: Error from server (NotFound): namespaces "cloud-aster" not found

INFO: 2018-08-21 14:16:51,782:Dart.py:426: Error from server (NotFound): namespaces "cloud-aster" not found

INFO: 2018-08-21 14:16:51,782:Dart.py:429: Error from server (NotFound): namespaces "cloud-aster" not found
INFO: 2018-08-21 14:16:51,783:Dart.py:453: Error from server (NotFound): namespaces "cloud-aster" not found.labs.teradata.com
INFO: 2018-08-21 14:16:51,783:Dart.py:460: []
INFO: 2018-08-21 14:16:51,783:Dart.py:461: [u'Error from server (NotFound): namespaces "cloud-aster" not found.labs.teradata.com']
INFO: 2018-08-21 14:16:51,783:Dart.py:299: Test Set: DockerHealth 
INFO: 2018-08-21 14:16:51,783:Dart.py:313: Running Testset: DockerHealth
INFO: 2018-08-21 14:16:51,784:Dart.py:319:         Running Test: ClusterHealth on Cluster dartpm18
INFO: 2018-08-21 14:16:51,784:Dart.py:322: [u'ClusterHealth', {u'CATEGORY': u'positive', u'NAME': u'ClusterHealth', u'CHECKLOG': u'no', u'COMPONENT': u'devops_installer/docker', u'LOCATION': u'devops_installer/docker/ClusterHealth', u'TIMEOUT': 1200, u'OWNER': u'divya.sivanandan', u'RESTART': u'yes'}, 0]
INFO: 2018-08-21 14:16:51,816:Dart.py:752: --------------------------------------------------
INFO: 2018-08-21 14:16:51,816:TestBase.py:42:           Using the following Test Parameters:
INFO: 2018-08-21 14:16:51,816:TestBase.py:43: --------------------------------------------------
INFO: 2018-08-21 14:16:51,816:TestBase.py:45:           CATEGORY : positive 
INFO: 2018-08-21 14:16:51,816:TestBase.py:45:           timeScale : 1 
INFO: 2018-08-21 14:16:51,817:TestBase.py:45:           NAME : ClusterHealth 
INFO: 2018-08-21 14:16:51,817:TestBase.py:45:           CHECKLOG : no 
INFO: 2018-08-21 14:16:51,817:TestBase.py:45:           filePath : devops_installer/docker 
INFO: 2018-08-21 14:16:51,817:TestBase.py:45:           logPath : /root/Dart 
INFO: 2018-08-21 14:16:51,817:TestBase.py:45:           COMPONENT : devops_installer/docker 
INFO: 2018-08-21 14:16:51,817:TestBase.py:45:           logTimestamp : 2018-08-21-141651 
INFO: 2018-08-21 14:16:51,818:TestBase.py:45:           LOCATION : devops_installer/docker/ClusterHealth 
INFO: 2018-08-21 14:16:51,818:TestBase.py:45:           TIMEOUT : 1200 
INFO: 2018-08-21 14:16:51,818:TestBase.py:45:           OWNER : divya.sivanandan 
INFO: 2018-08-21 14:16:51,818:TestBase.py:45:           RESTART : yes 
INFO: 2018-08-21 14:16:51,818:TestBase.py:46: --------------------------------------------------
INFO: 2018-08-21 14:16:51,838:Kubernetes.py:82: Executing command [asterdart0078.labs.teradata.com]: kubectl get pods --namespace=cloud-aster
INFO: 2018-08-21 14:16:52,214:TestBase.py:170: 
INFO: 2018-08-21 14:16:52,214:TestBase.py:172: No Aster cluster installed on this vms. Returns default values
ERROR: 2018-08-21 14:16:52,228:Dart.py:772: unsupported operand type(s) for +: 'NoneType' and 'str'
ERROR: 2018-08-21 14:16:52,228:Dart.py:773: The test raised an Unexpected Exception!
ERROR: 2018-08-21 14:16:52,228:Dart.py:774: (<type 'exceptions.TypeError'>, TypeError("unsupported operand type(s) for +: 'NoneType' and 'str'",), <traceback object at 0x7f66dc0b9680>)
ERROR: 2018-08-21 14:16:52,229:Dart.py:775: Traceback (most recent call last):
  File "/root/Dart/Dart.py", line 756, in runTest
    testInstance = test(self.cfgJson, testParam)
  File "/root/Dart/testsrc/devops_installer/docker/KubernetesTestBase.py", line 25, in __init__
    self.ansibleSetupDir = self.buildLoc + "/asterK8sInstaller/kubernetes/multinode/vrealize/ansible-remote"
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'

INFO: 2018-08-21 14:16:52,229:Dart.py:781: The test ClusterHealth has FAILED
ERROR: 2018-08-21 14:16:52,229:Dart.py:325: unsupported operand type(s) for +: 'NoneType' and 'str'
ERROR: 2018-08-21 14:16:52,230:Dart.py:327: (<type 'exceptions.TypeError'>, TypeError("unsupported operand type(s) for +: 'NoneType' and 'str'",), <traceback object at 0x7f66dc0d6c20>)
ERROR: 2018-08-21 14:16:52,230:Dart.py:328: Traceback (most recent call last):
  File "/root/Dart/Dart.py", line 323, in __init__
    self.runTest(testset, test[0], test[1], test[2])
  File "/root/Dart/Dart.py", line 756, in runTest
    testInstance = test(self.cfgJson, testParam)
  File "/root/Dart/testsrc/devops_installer/docker/KubernetesTestBase.py", line 25, in __init__
    self.ansibleSetupDir = self.buildLoc + "/asterK8sInstaller/kubernetes/multinode/vrealize/ansible-remote"
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'

INFO: 2018-08-21 14:16:52,231:Dart.py:329: Continuing with the remaining tests!
INFO: 2018-08-21 14:16:52,231:Dart.py:334: Test Set : DockerHealth Total Execution Time: 0 
